README.md

```diff
@@ -1,9 +1,9 @@
-# fornax-demo-notebooks
```

Comment:
I'm fairly certain that we still need a top level heading, but we can try and see and fix only if it causes problem.